policy/validation.rego

```diff
@@ -4,6 +4,10 @@ default deny = false
 
 allowed_types := ["kubernetes"]
 
+is_pipeline {
```

Comment:
The rego policy can be simplified with the removal of some of the business logic and using multiple deny statements.  Also during testing if the `kind` attribute is missing entirely validation seems to let this through.

```
is_pipeline_empty {
  input.kind == ""
}

is_valid {
    allowed_types[_] == input.type
}

deny[msg] {
  is_pipeline
  not is_valid
  msg = sprintf("type '%v' is not supported", [input.type])
}


deny[msg] {
	is_pipeline_empty
  msg = sprintf("pipeline: '%v' is not a valid kind", [input.kind])
}
```

Reply:
I didnt consider the user error message. Ignore this.